### DISCUSSION TITLE
'make gpu' results in error

### DISCUSSION THREAD

**Discussion by Parandrus** [Harvard-Neutrino/prometheus#65] [Help needed]
I am trying to compile the CUDA PPC code on a cluster. When running 'make gpu' in the PPC_CUDA directory, I get the following error: 

```bash
$ make gpu
nvcc ppc.cu -Xptxas=-v -arch=sm_50 -O2 --use_fast_math --compiler-options=-O2,--fast-math -o ppc 2>&1 | grep -v assuming
pro.cu(58): error: identifier "int_as_float" is undefined
    return int_as_float(tmp|0x3f800000)-1.0f;
           ^

1 error detected in the compilation of "ppc.cu".
```

`int_as_float` is used outside the scope of the `#ifdef XCPU`, but it seems like 'pro.cu' is included after `#define XCPU` in 'ppc.cu', so I don't see why there should be any errors. Adding `#define XCPU` to the top of `pro.cu` introduces more errors.

**Comment by wesmail** [comment]
If you are using Docker/Singularity container. Make sure that `nvcc` is available
